inc/html.class.php

```diff
@@ -1910,15 +1910,15 @@ static function footer($keepDB = false)
          ? $CFG_GLPI['founded_new_version']
          : '';
       if (!empty($foundedNewVersion) && version_compare($currentVersion, $foundedNewVersion, '<')) {
-         $twig_vars["latest_version"] = "<a href='http://www.glpi-project.org' target='_blank' title=\""
-            . __s('You will find it on the GLPI-PROJECT.org site.') . "\"> "
+         $twig_vars["latest_version"] = "<a href='https://www.itsm-ng.org/' target='_blank' title=\""
+            . __s('You will find it on the ITSM-NG.org site.') . "\"> "
```

Comment:
ca pointe sur le site en majuscule ?

Reply:
No, and if you use ITSM-NG.org on your browser it's work :smile: